Artifacts/windows-jhipster/Artifactfile.ps1

```diff
@@ -0,0 +1,294 @@
+﻿<##################################################################################################
```

Comment:
May want to keep the same name as the other startChocolatey script files so that we catch them when we need changes